src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -3028,9 +3028,9 @@ def import_ssl_cert(cmd, resource_group_name, name, key_vault, key_vault_certifi
         if kv_subscription.lower() != subscription_id.lower():
             diff_subscription_client = get_mgmt_service_client(cmd.cli_ctx, ResourceType.MGMT_APPSERVICE,
                                                                subscription_id=kv_subscription)
-            ascs = diff_subscription_client.app_service_certificate_orders.list()
+            ascs = diff_subscription_client.app_service_certificate_orders.list(api_version='2022-09-01')
         else:
-            ascs = client.app_service_certificate_orders.list()
+            ascs = client.app_service_certificate_orders.list(api_version='2022-09-01')
```

Comment:
It is not recommended to hard code the api_version in custom.py file. You can add ad-hoc api-version in https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/azure/cli/core/profiles/_shared.py#L432.

Hi @jiasli, do you have any suggestions for this?

Reply:
Thanks for your comment @yanzhudd. I just addressed it. Can you please review the PR again? Thank you!

Reply:
Suggest defining a reusable variable for `api_version='2022-09-01'`
```
 ResourceType.MGMT_APPSERVICE: {
        # src/azure-cli/azure/cli/command_modules/appservice/custom.py:3033
        'app_service_certificate_orders': '2022-09-01'
    },
```
`# src/azure-cli/azure/cli/command_modules/appservice/custom.py:3033` so the referenced code link can point to reusable variable

Reply:
Ok I have addressed your comment. Can you please review it again @zhoxing-ms and merge if it looks good? Thank you!